content/root/platform-customization.textile

```diff
@@ -57,29 +56,31 @@ Note that this does not prevent customers outside of your chosen region from con
 
 If you are interested in applying processing or storage constraints to regions other than Europe or the US, then please speak to your customer success manager.
 
-To implement regional restrictions at the application level, you must specify the required environment after configuring your account for regional routing and storage. Specify the required environment: @eu@ or environment: @us@ in the Ably client options ( "Rest SDK":/docs/api/rest-sdk#client-options and "Realtime SDK":/docs/api/realtime-sdk#client-options ) when you are initiating the connection, from both the publisher's and subscriber's side.
+To implement regional restrictions at the application level, you must specify the required endpoint after configuring your account for regional routing and storage. Specify the required region in your endpoint. Once configured by Ably, you specify this endpoint in the Ably client options ( Rest SDK and Realtime SDK) when you are initiating the connection, from both the publisher's and subscriber's side. Your endpoint would be similar to what's shown in the examples below:
+
+* for `eu`, `example-eu` would have the url `example-eu.realtime.ably.net or `example-eu.rest.ably.net` and will be configured to keep the data within the EU.
+* for `us`, `example-us` would have the url `example-us.realtime.ably.net` or `example-us.rest.ably.net` and will be configured to keep the data within the US.
 
 *Note*: This approach does not apply regional restrictions when you are using a persistent history feature. For implementing geolocation restrictions on history, you would need to set a placement constraint to ensure data stays in the EU or any designated region.
 
-h2(#setting-up-a-custom-environment). Setting up a custom environment
+h2(#setting-up-a-custom-endpoint). Setting up a custom endpoint
 
-To use a custom environment, perform the following steps:
+To use a custom endpoint, perform the following steps:
 
 h3. Decide whether to use an Ably subdomain, or your own domain
 
 Ably client libraries connect to two endpoints by default:
 
-- @realtime.ably.io@ - this endpoint is used for all WebSocket and other realtime protocol connections from the Ably client libraries.
-- @rest.ably.io@ - this endpoint is used for all REST-based requests and is optimized for all standard HTTP-based request types including all "REST requests":/docs/api/rest-api, "authentication requests":/docs/auth, and Comet and JSONP fallback when WebSockets cannot be used.
+- @realtime.ably.net@ - this endpoint is used for all WebSocket and other realtime protocol connections from the Ably client libraries.
+- @rest.ably.net@ - this endpoint is used for all REST-based requests and is optimized for all standard HTTP-based request types including all "REST requests":/docs/api/rest-api, "authentication requests":/docs/auth, and Comet and JSONP fallback when WebSockets cannot be used.
```

Comment:
Probably want to rephrase this suggestion but basically we only have a single endpoint now, we do not have a separate one for REST.

```suggestion
- @main.realtime.ably.net@ - this endpoint is used by both WebSocket and other realtime protocol connections from the Ably client libraries and all REST-based requests and is optimized for all standard HTTP-based request types including all "REST requests":/docs/api/rest-api, "authentication requests":/docs/auth, and Comet and JSONP fallback when WebSockets cannot be used.
```